Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -67,11 +67,24 @@
     CKFPerformanceConfig(ptMin=400.0 * u.MeV, nMeasurementsMin=6),
     outputDirRoot=outputDir,
 )
+s.addAlgorithm(
+    acts.examples.TrackSelector(
+        level=acts.logging.INFO,
+        inputTrackParameters="fittedTrackParameters",
+        outputTrackParameters="trackparameters",
+        outputTrackIndices="outputTrackIndices",
+        removeNeutral=True,
+        absEtaMax=2.5,
+        loc0Max=4.0 * u.mm,  # rho max
+        ptMin=500 * u.MeV,
+    )
+)
 s = addVertexFitting(
     s,
     field,
-    vertexFinder=VertexFinder.Truth,
+    vertexFinder=VertexFinder.Iterative,
```

Comment:
Can we try the AMVF here? (This is what ATLAS uses now)

Reply:
sure we can have AMVF